src/vegasflow/__init__.py

```diff
@@ -4,5 +4,6 @@
 # Expose the main interfaces
 from vegasflow.vflow import VegasFlow, vegas_wrapper
 from vegasflow.plain import PlainFlow, plain_wrapper
+from vegasflow.stratified import StratifiedFlow, stratified_wrapper
```

Comment:
Is this the correct way to modify __init__.py?

Reply:
Yes.

Reply:
Thanks!

Reply:
I am also writing an example similar to `examples/simgauss_tf.py` using the class `StratifiedFlow` and the same line gives me a `ModuleNotFoundError`. Is this because this branch is not yet been merged to the master branch?

Reply:
Did you install this branch with `python setup.py develop`? If not you can to run this command or reinstall the branch.

Reply:
I mean, you have to uninstall the official vegasflow with `pip uninstall vegasflow`, then enter this branch and run the command above. The `develop` option create links to the source files so your edits will automatically be available.

Reply:
I got it. Now it works fine. I'll update the example shortly. Thanks!